include/cooperative_perception/angle.hpp

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2022 Leidos
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ANGLE_HPP
+#define ANGLE_HPP
+
+#include <complex>
+#include <math.h>
+#include <units.h>
+
+/**
+ * @brief Angle type for representing and manipulating angles
+ */
```

Comment:
This should go over the `Angle` class body, not the `cooperative_perception` namespace.